firebase-config/ktx/ktx.gradle

```diff
@@ -49,6 +49,7 @@ dependencies {
     implementation project(':firebase-config')
     implementation project (':firebase-abt')
     implementation 'androidx.annotation:annotation:1.1.0'
+    implementation 'com.google.android.gms:play-services-tasks:17.0.0'
```

Comment:
Is this dependency needed? The only new import is in `RemoteConfig.kt` and doesn't seem related to that library.

Reply:
It was being used for the second proposal. I forgot to remove the dependency when I removed the function. But it's done now, thanks